content/blog/LFX-Mentorship-Blog.md

```diff
@@ -0,0 +1,425 @@
+
+- Set the env variable
+
+```bash
+export GOOGLE_APPLICATION_CREDENTIALS="<path-to-service-token-json-file>"
+```
+
+- Command used to run tremor:
+
+```bash
+ TREMOR_PATH=tremor-script/lib target/debug/tremor server run -f outbound.trickle inbound.trickle test.yaml | jq .
+```
+For a detailed guide on the operations that can be performed, refer the [docs](https://docs.tremor.rs/artefacts/offramps/#gcs).
+
+### Google Cloud Pub/sub
+
+Google Cloud Pub/Sub guarantees delivery of all messages, whether low throughput or high throughput, so there should be no concern about messages being lost.
+
+Pub/Sub guarantees at-least-once message delivery, which means that occasional duplicates are to be expected since we acknowledge the messages once they are received.
+
+The following files are required:
+
+**outbound trickle:** 
+
+```trickle
+select event from in into out
+```
+
+**inbound.trickle:**
+
+```trickle
+select {"data": event, "meta": $} from in into out;
+```
+
+**test.yaml:**
+
+```yaml
+offramp:
+  - id: stdout
+    type: stdout
+    codec: json
+    config:
+  - id: gpub
+    type: gpub
+    codec: json
+    postprocessors:
+      - gzip   
+    linked: true 
+    config:
+      pem: <path-to-pem-file>
+onramp:
+  - id: stdin
+    type: file
+    codec: json
+    config:
+      source: /dev/stdin  
+  - id: gsub
+    type: gsub
+    codec: json  
+    preprocessors:
+      - gzip
+    config:
+      pem: <path-to-pem-file>
+      subscription: '<name-of-subscription>'
+binding:
+  - id: example
+    links:
+      '/onramp/stdin/{instance}/out':
+        - '/pipeline/outbound/{instance}/in'
+      '/pipeline/outbound/{instance}/out':
+        - '/offramp/stdout/{instance}/in'
+        - '/offramp/gpub/{instance}/in'
+      '/offramp/gpub/{instance}/out':
+        - '/pipeline/inbound/{instance}/in'
+      '/pipeline/inbound/{instance}/out':
+        - '/offramp/stdout/{instance}/in'
+      '/onramp/gsub/{instance}/out':
+        - '/pipeline/inbound/{instance}/in'
+mapping:
+  "/binding/example/passthrough":
+    instance: "passthrough"
+```
+
+Supported preprocessors, that can be configured in yam file can be found here: [preprocessors](https://docs.tremor.rs/artefacts/preprocessors/)
+
+Supported postprocessors and more about it: [postprocessors](https://docs.tremor.rs/artefacts/postprocessors/)
+
+Supported codecs, that can be configured in yam file can be found here: [codecs](https://docs.tremor.rs/artefacts/codecs/)
+
+![Tremor Dot Diagrm](/img/blog/LFX-Mentorship/dot-diagram.png)
+
+- Create a topic using the following `gcloud` command:
+
+```bash
+gcloud pubsub topics create <`topic-name`>
+```
+
+- Set the env variable
+
+```bash
+export GOOGLE_APPLICATION_CREDENTIALS="<path-to-service-token-json-file>"
+```
+
+- Command used to run tremor:
+
+```bash
+TREMOR_PATH=tremor-script/lib target/debug/tremor server run -f outbound.trickle inbound.trickle test.yaml | jq .
```

Comment:
This might need some explanation setting tremor path like this requires to be in the source directory the post should probably not make that assumption but rather that the user installed the binaries (same goes for the first invocation)

Reply:
Okay do I then remove this `TREMOR_PATH` from the command assuming the user would have it set?